postreise/analyze/generation/costs.py

```diff
@@ -0,0 +1,51 @@
+import numpy as np
+import pandas as pd
+from numpy.polynomial.polynomial import polyval
+
+from postreise.analyze.check import (
+    _check_gencost,
+    _check_scenario_is_in_analyze_state,
+    _check_time_series,
+)
+
+
+def calculate_costs(scenario=None, pg=None, gencost=None, decommit=False):
+    """Calculate individual generator costs at given powers. If decommit is
+    True, costs will be zero below the decommit threshold (1 MW).
+    Either ``scenario`` XOR (``pg`` AND ``gencost``) must be specified.
+
+    :param powersimdata.scenario.scenario.Scenario scenario: scenario to analyze.
+    :param pandas.DataFrame pg: Generation solution data frame.
+    :param pandas.DataFrame gencost: cost curve polynomials.
+    :param boolean decommit: Whether to decommit generator at very low power.
+    :return: (*pandas.DataFrame*) -- data frame of costs.
+    """
+    decommit_threshold = 1  # MW
```

Comment:
Is it worthwhile to make it a keyword argument so that one could modify it if necessary?

Reply:
Done.